cli/src/formatter.rs

```diff
@@ -166,6 +166,32 @@ impl FormatterFactory {
     }
 }
 
+pub struct SilentFormatter;
```

Comment:
nit: `io::sink()` can also be used.
https://doc.rust-lang.org/std/io/fn.sink.html

Reply:
I suppose that works only if we return a `Box<dyn Write>` instead?

Reply:
If we decided to return `io::Write` object, `Box<dyn>` or `Either` would be needed. For `Box<dyn Fromatter>, the inner `io::Write` object doesn't matter.

Reply:
Done.